code/game/gamemodes/wizard/wizard.dm

```diff
@@ -246,22 +246,6 @@
 		qdel(spell_to_remove)
 		mind.spell_list -= spell_to_remove
 
-/*Checks if the wizard can cast spells.
-Made a proc so this is not repeated 14 (or more) times.*/

```

Comment:
To be fair this comment is correct, the proc isn't repeated 14 or more times :smile: